rules/index.md

```diff
@@ -1,14 +1,14 @@
 ---
```

Comment:
I'd rather we not ship this page updates until we actually ship the rules, so it's obvious we've not pushed them out yet. Also means we don't ship a known-bad link

Reply:
If you're happy to be the one to handle this, then i'm ok with that :+1: